.github/workflows/trivy.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@062f2592684a31eb3aa050cc61e7ca1451cecd3d
         with:
-          image-ref: 'ghcr.io/rancher-sandbox/rancher-turtles-amd64:${{ github.sha }}'
+          image-ref: 'ghcr.io/rancher/turtles-amd64:${{ github.sha }}'
```

Comment:
CI output:

`* docker error: unable to inspect the image (ghcr.io/rancher/turtles-amd64:ac8e9b0ddc6a2e5deaaadc4721638aa1b1d4b05d): Error response from daemon: No such image: ghcr.io/rancher/turtles-amd64:ac8e9b0ddc6a2e5deaaadc4721638aa1b1d4b05d`

looks like this will fail until we have a new image built and pushed to new` ghcr.io/rancher/turtles` location. I think we could revert this change for now, merge this PR (which should built image with new location) without trivy update, open follow-up PR to update it?

Reply:
Forgot to update repo name in Makefile which is used to build the docker image, this is fixed now